### DISCUSSION TITLE
How to pass custom ssh port?

### DISCUSSION THREAD

**Discussion by AnoRebel** [coollabsio/coolify#1949] [Q&A] [answered]
I'm trying to setup coolify on my own server but my ssh runs on a custom port and not the default port, 22.
I checked the documentation on OpenSSH in the coolify website but I dont see an area to pass the custom port. 
Am I missing something? If not, any help on how I can go about this would be really great?
Thanks in advance.

**Comment by andrasbacsai** [comment]
After installing Coolify, you can register your first user and then modify the server's port from the default port 22 on `localhost` to a custom port of your choice.

**Reply by eduludi** [reply]
Can you please elaborate on this? Where is it possible to do this? Form Coolify backend UI or a configuration file in the server? Thanks!

**Reply by peaklabs-dev** [reply, marked as answer]
@AnoRebel and @eduludi The custom port can be set after the admin user is registered in the server settings: 
<img width="1327" alt="image" src="https://github.com/user-attachments/assets/155d9a15-9e2e-40b3-a2a6-8c1cfe465dc7">

**Reply by wei** [reply]
Hiii I was hoping to go through the onboarding but I got stuck on the server selection screen. 

I followed the above, skipped onboarding to add the server manually but after going back to onboarding, it still doesn't allow me to go past the server selection screen. It would be amazing if we could continue with the boarding after a server has been added~

**Comment by AnoRebel** [comment]
But I can't get the coolify to start, and I thought the error was it failing to connect to ssh.
Do I have to change my ssh port back to the default just for the install.?

**Reply by Niskeletor** [reply]
@AnoRebel  In the right side you can find a section about skip configuration, press on that option and then you can config custom ssh port

**Reply by AnoRebel** [reply]
In the docs?

**Reply by peaklabs-dev** [reply]
@AnoRebel Just install using the script and then register your user, and change the setting above.

**Reply by AnoRebel** [reply]
> @AnoRebel Just install using the script and then register your user, and change the setting above.

I understand this part, my question now was/is if i can pass a custom port in a config or something during/before installation, not after installation

**Reply by peaklabs-dev** [reply]
@AnoRebel I added a feature reuest for this: https://github.com/coollabsio/coolify/discussions/3205.
